examples/a_treeple/README.txt

```diff
@@ -0,0 +1,6 @@
+.. _treeple:
+
+Treeple
+-------
+
+Examples concerning how to use treeple as hypothesis test tools.
```

Comment:
Perhaps we can also specify an order for the tutorials. Namely posterior estimates seems like a logical first choice? Wdyt?